gourmet/plugin_gui.py

```diff
@@ -71,20 +81,15 @@ def make_treeview (self, plugin_list):
         text_renderer = Gtk.CellRendererText()
         text_renderer.set_property('wrap-width',350)
         plugin_col = Gtk.TreeViewColumn('Plugin',text_renderer)
-        def data_fun (col,renderer,mod,itr):
-            plugin_set = mod[itr][1]
-            renderer.set_property('markup',
-                                  ('<b>'+escape(plugin_set.name)+ '</b>' +
-                                   '\n<span size="smaller"><i>' + escape(plugin_set.comment) + '</i></span>')
-                                  )
-        plugin_col.set_cell_data_func(text_renderer,data_fun)
+        plugin_col.set_cell_data_func(text_renderer, self.data_fun)
         plugin_col.set_property('expand',True)
         plugin_col.set_property('min-width',250)
         tv.append_column(plugin_col)
         tv.append_column(active_col)
         ls = self.make_list_store(plugin_list)
         tv.set_model(ls)
-        sw = Gtk.ScrolledWindow(); sw.set_policy(Gtk.PolicyType.NEVER,Gtk.PolicyType.AUTOMATIC)
+        sw = Gtk.ScrolledWindow()
+        sw.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.NEVER)
```

Comment:
Is this just whitespace? I know it's bad, but that can be done automatically later on